Setzen der Repository-URL fuer nachtraegliche Paket-Installationen
Wir setzen die URL absolut ("=" anstelle von ":="), um sicherzustellen, dass die Opennet-Versionszahlen sichtbar sind
Index: on_firmware/openwrt/include/version.mk
===================================================================
--- on_firmware.orig/openwrt/include/version.mk	2014-12-05 22:54:37.568003627 +0100
+++ on_firmware/openwrt/include/version.mk	2014-12-05 22:58:11.756069434 +0100
@@ -7,6 +7,7 @@
 
 # Substituted by SDK, do not remove
 # REVISION:=x
+include $(INCLUDE_DIR)/opennet-version.mk
 
 PKG_CONFIG_DEPENDS += \
 	CONFIG_VERSION_NUMBER \
